Design/Workbench/design-projectsaving.md

```diff
@@ -175,7 +175,7 @@ Similarly with the C++ area of serialisation exposing most of the required detai
 
 Implementation of save workspaces and linking to saved scripts
 --------------------------------------------------------------
-Utilizing the already implemented SaveNexusProcessed and SaveMD algorithms, the suggestion is to implement a python version of how MantidPlot currently implements saving workspaces. Call SaveNexusProcessed for most workspaces and SaveMD for EventWorkspaces. Saving a copy of the names of the saved workspace .nxs files in a JSON format in the same file as the output from the serialization of the windows, it may look like this:
+Utilizing the already implemented SaveNexusProcessed, SaveMD, and SaveMask algorithms, the suggestion is to implement a python version of how MantidPlot currently implements saving workspaces. Call SaveNexusProcessed for most workspaces, SaveMD for EventWorkspaces, and SaveMask for MaskedWorkspaces. Saving a copy of the names of the saved workspace .nxs files in a JSON format in the same file as the output from the serialization of the windows, it may look like this:
```

Comment:
I don't think we should make a distinction for `MaskWorkspace`. If it's a workspace then we should just use `SaveNexusProcessed`.

Reply:
I agree. @SamJenkins1 perhaps we should look at that as a prerequisite?

Reply:
I agree. @Pasarus perhaps we should look at that as a prerequisite?